hubspot/plugin.py

```diff
@@ -1515,6 +1526,14 @@ def hubspot_list_deals(
     deals = []
     for item in data["results"]:
         properties = item["properties"]
+        property_values = {key: val for key, val in properties.items() if val}
```

Comment:
if Val here would do strange things like remove properties with the value 0 or empty string

Reply:
(actually, why do you need this?)